patterns/generic-emergency.php

```diff
@@ -0,0 +1,487 @@
+<?php
+/**
+ * Title: Generic Emergency
+ * Slug: bcgov-wordpress-block-theme-child-theme-emergency-info/generic_emergency
+ * Categories: eibc_event, eibc_generic_emergency
```

Comment:
```suggestion
 * Categories: eibc_event, eibc_generic
```
This is the slug for the hazard type on prod.

Reply:
well that a little generic dont ya think?

Reply:
Yeah it's not a hazard type that's really supposed to be used but it's there in case some unknown hazard happens that isn't already covered by another hazard_type term.